Model/Reclaim.php

```diff
@@ -106,6 +111,52 @@ public function product($quote_id, $item_id) {
         return $response;
     }
 
+    /**
+    * @return mixed
+    */
+    public function backinstock($product_id)
+    {
+        if (!$product_id){
+            throw new NotFoundException(_('A product id is required'));
+        }
+        $product = $this->_objectManager->create('Magento\Catalog\Model\Product')->load($product_id);
+
+        if (!$product){
+            throw new NotFoundException(_('A product was not found'));
+        }
+
+        $productId = $product->getId();
+
+        $response = array(array(
+            'id' => $productId,
+            'title' => $product->getName(),
+            'price' => $product->getPrice(),
+            'available' => true,
+            'inventory_quantity' => $this->_stockItem->getStockQty($productId),
+            'inventory_policy' => $this->_getStockItem($productId)
+        ));
+        // check to see if the product has variants, if it doesn't just return the product information
+        try {
+            $_children = $product->getTypeInstance()->getUsedProducts($product);
+            // throws a fatal error, so catch it generically and return
+        } catch (\Error $e) {
+            return $response;
+        }
+        
+        foreach ($_children as $child){
+            $response['variants'][] = array(
+                'id' => $child->getId(),
```

Comment:
And here too please:
```suggestion
                'id' => $child->getSku(),
```